src/integrationtests/java/com/aws/greengrass/integrationtests/NucleusLaunchUtils.java

```diff
@@ -141,9 +142,7 @@ void startNucleusWithConfig(NucleusLaunchUtilsConfig config) throws InterruptedE
             }
             syncHandler.setOverallSyncStrategy(syncStrategy);
             isSyncMocked.set(true);
-            shadowManager.startSyncingShadows(ShadowManager.StartSyncInfo.builder().startSyncStrategy(true).build());
```

Comment:
i think this was the cause of some flakiness, this clears the sync queue and puts full sync, on top of the full sync that already happens when shadow is started up